lib/job/remove.ts

```diff
@@ -5,6 +5,6 @@ import { Job } from '.';
  * @name Job#remove
  * @function
  */
-export const remove = function(this: Job) {
+export const remove = async function(this: Job) {
```

Comment:
no need for async or? instead you could add a `:Promise<..>` as return value

Reply:
Typescript/eslint enforces that that "async" - "Promise" come in tandem.
You can't have one without the other.